.github/workflows/trivy-scan.yaml

```diff
@@ -0,0 +1,31 @@
+name: Trivy Scan
+
+on:
+  push:
+    # Run on main and supported release branches
+    branches:
+      - main
+      - release-1.23
+      - release-1.22
+      - release-1.21
+  # Run weekly
+  schedule:
+    - cron: '0 12 * * 1'
+  # Allow manual runs
+  workflow_dispatch:
+
+jobs:
+  trivy-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: aquasecurity/trivy-action@master
```

Comment:
any particular reason to use `master` vs the latest tagged release (https://github.com/aquasecurity/trivy-action/releases/tag/0.8.0)?